### PR TITLE
version updates and misc

### PR DESCRIPTION
**Pull request by jcmrva**
I was initially going to change the Forge page to mention that it's been superseded but IMO it makes more sense to just remove it. I can restore that if preferred.

I wasn't sure about Waypoint.

I noticed FSharp.Analyzers.SDK currently targets net5.0, references in the docs here say netstandard2.0. Can we bump that across the board if nothing needs to support .NET Framework? Same with Fornax.

### CODE REVIEW

File: content/Editors/Code/getting_started.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ menu_order: 2
 
 * Visual Studio Code - it is a lightweight but powerful source code editor which runs on your desktop and is available for Windows, macOS and Linux created by Microsoft. For detailed documentation of editor, getting-started guides and more visit [official documentation](https://code.visualstudio.com/docs).
 
-* F# - it is a mature, open source, cross-platform, functional-first programming language. It empowers users and organizations to tackle complex computing problems with simple, maintainable and robust code. Ionide supports any version of F# >= 3.0 but we do recommend using F# 4.1. Detailed installation instructions can be found of F# Software foundation webpage - for [Windows](http://fsharp.org/use/windows/), [MacOS](http://fsharp.org/use/mac/), and [Linux](http://fsharp.org/use/linux/)
```
Comment (jcmrva) [inline content/Editors/Code/getting_started.md:12]: With FSAC now requiring net6.0, does it make sense to mention older versions of the language?

File: content/Editors/Code/getting_started.md [hunk 2]
```diff
@@ -9,9 +9,9 @@ menu_order: 2
...
 
-* .Net Core SDK - .Net Core is lightweight, cross platform, modern implementation of .Net Framework. We strongly recommend installing it since some advanced Ionide features such as debugging and project scaffolding depends on SDK and `dotnet` CLI tooling even if your application is targetting Full Framework. For detailed instructions on installing .Net Core visit [official step-by-step installation guide](https://www.microsoft.com/net/core)
+* .NET SDK - .NET is lightweight, cross platform, modern implementation of .NET Framework. We strongly recommend installing it since some advanced Ionide features such as debugging and project scaffolding depends on SDK and `dotnet` CLI tooling even if your application is targetting Full Framework. For detailed instructions on installing .NET visit [official step-by-step installation guide](https://dotnet.microsoft.com/download)
 
 * VSCode C# plugin (optional) - Ionide's debugging capabilities relies on the debugger provided by Omnisharp team. To get it install [C# extension from VSCode marketplace](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
```
Comment (jcmrva) [inline content/Editors/Code/getting_started.md:17]: Unsure if the MsBuild info below needed to be updated.

File: content/Editors/Code/getting_started.md (hunk 1 above)
Comment (baronfel) [reply content/Editors/Code/getting_started.md:12]: Not at all, this is a good change.

**Review by baronfel — APPROVED** [review]

**Comment by jcmrva** [comment]
@baronfel Just noticed when running locally it moves the Tools section to the bottom. No idea why...

File: content/Editors/Code/getting_started.md (hunk 2 above)
Comment (baronfel) [reply content/Editors/Code/getting_started.md:17]: The support for old-style projects got added back in recently, actually. It's very minimal support, though. We should move to suggesting VS 2022 as the install mechanism for it.

**Comment by baronfel** [comment]
Thanks for updating these! Any last things you want to cover before I merge?

**Comment by jcmrva** [comment]
Just what's in the original comment, but I suppose those can wait.

**Comment by baronfel** [comment]
ah, taking your questions one by one then:
* forge removal is fine
* waypoint can stay as it is, we'll have to do a separate pass on it to freshen it up
* the Analyzer SDK will bump to net6.0 with the next update to it, so aligning everything to net5.0 now will make it easier to update those references to net6.0, if you wouldn't mind doing that

**Comment by baronfel** [comment]
Thanks!

**Pull request merged**